napari/components/_window/view.py

```diff
@@ -21,15 +22,22 @@ def __init__(self, viewer, show=True):
         self._qt_window = QMainWindow()
         self._qt_center = QWidget()
         self._qt_window.setCentralWidget(self._qt_center)
+        self._qt_window.setWindowTitle('napari')
         self._qt_center.setLayout(QHBoxLayout())
         self._status_bar = self._qt_window.statusBar()
         self._status_bar.showMessage('Ready')
 
         self._help = QLabel('')
         self._status_bar.addPermanentWidget(self._help)
 
+        self._status_bar.setStyleSheet('QStatusBar { background: %s; color: %s}' % 
```

Comment:
Can this go to the `stylesheet.qss` too?

Reply:
for everything in `_window` that doesn't appear to work -- the contents of `stylesheet.qss` is currently applied to the viewer (in `_viewer/view/main`), which is a component of the `window`

what i thought might work is to call `setStyleSheet` on the `_qt_window` here, which would achieve the same goal of doing it in just one place, but that weirdly only applies some of the styles

maybe we can pair on this, obviously better to do it all in one place!

Reply:
ah ok - curious what @kne42 thinks. We could just leave it for now. We will likely be refactoring the `QtViewer`, `ViewerApp`, and `Window` stuff pretty soon anyway